libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2089,6 +2089,81 @@ MAV_RESULT GCS_MAVLINK::handle_command_preflight_set_sensor_offsets(const mavlin
     return MAV_RESULT_ACCEPTED;
 }
 
+bool GCS_MAVLINK::calibrate_gyros()
+{
+    AP::ins().init_gyro();
+    if (!AP::ins().gyro_calibrated_ok_all()) {
+        return false;
+    }
+    AP::ahrs().reset_gyro_drift();
+    return true;
+}
+
+MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration_baro()
+{
+    gcs().send_text(MAV_SEVERITY_INFO, "Updating barometer calibration");
+    AP::baro().update_calibration();
```

Comment:
Sub didn't do this, it did a full calibration. It's already overriding this method but still calling this.